### PR TITLE
New function to retry transfers

### PR DESCRIPTION
**Pull request by javiserrano**

### CODE REVIEW

File: include/megaapi.h [hunk 1]
```diff
@@ -6551,6 +6551,22 @@ class MegaApi
         void cancelTransfer(MegaTransfer *transfer, MegaRequestListener *listener = NULL);
 
         /**
+         * @brief Retry a transfer
+         *
+         * This function allows to start a transfer based on a MegaTransfer object. It can be used,
+         * for example, to retry transfers that finished with an error. To do it, you can retain the
+         * MegaTransfer object in onTransferFinish and use it later with this function.
```
Comment (sergiohs84) [inline include/megaapi.h]: You can do so by calling \c MegaTransfer::copy()

File: src/megaapi_impl.cpp [hunk 2]
```diff
@@ -5688,7 +5688,36 @@ void MegaApiImpl::startStreaming(MegaNode* node, m_off_t startPos, m_off_t size,
 	transfer->setEndPos(startPos + size - 1);
 	transfer->setMaxRetries(maxRetries);
 	transferQueue.push(transfer);
-	waiter->notify();
+    waiter->notify();
```
Comment (sergiohs84) [inline src/megaapi_impl.cpp]: Indentation?

File: include/megaapi.h (hunk 1 above)
Comment (sergiohs84) [inline include/megaapi.h]: I would add:
`You can do so by calling \c MegaTransfer::copy()`

File: src/megaapi_impl.cpp (hunk 2 above)
Comment (javiserrano) [inline src/megaapi_impl.cpp]: Not in the new code, but I will fix the lines above

File: include/megaapi.h (hunk 1 above)
Comment (javiserrano) [inline include/megaapi.h]: I agree

**Pull request merged**